src/ALAC/alac_codec.h

```diff
@@ -28,6 +28,18 @@
 
 #include <stdint.h>
 
+#if HAVE_STDBOOL_H
+#include <stdbool.h>
+#endif
+
+#if !defined(__cplusplus) && !defined(__bool_true_false_are_defined)
+typedef enum
+{
+	false = 0,
+	true = 1
+} bool ;
+#endif
```

Comment:
nix this entire lot and just include `<stdbool.h>` in the headers/files that need it. It's 2024, if you dont have a C99 compiler, I'm sorry for you.